app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt

```diff
@@ -271,35 +285,56 @@ private fun PlantDetailsContent(
     }
 }
 
+@OptIn(ExperimentalGlideComposeApi::class)
 @Composable
 private fun PlantImage(
     imageUrl: String,
     imageHeight: Dp,
     modifier: Modifier = Modifier,
     placeholderColor: Color = MaterialTheme.colors.onSurface.copy(0.2f)
 ) {
-    val painter = rememberAsyncImagePainter(
-        model = ImageRequest.Builder(LocalContext.current)
-            .data(data = imageUrl)
-            .crossfade(true)
-            .build()
-    )
-
-    Image(
-        painter = painter,
-        contentScale = ContentScale.Crop,
-        contentDescription = null,
-        modifier = modifier
+    var isLoading by remember { mutableStateOf(true) }
+    Box(
+        modifier
             .fillMaxWidth()
-            .height(imageHeight)
-    )
-
-    if (painter.state is AsyncImagePainter.State.Loading) {
-        Box(
+            .height(imageHeight)) {
+        if (isLoading) {
+            Box(
+                Modifier
+                    .fillMaxSize()
+                    .background(placeholderColor)
+            )
+        }
```

Comment:
Glide doesn't have some sort of placeholder functionality built in?

Reply:
Ah - that just got added recently so will update this PR

Reply:
Looks like it's not released yet. Will add a comment to update it when that feature is available on maven.

Reply:
Sounds good!